### ISSUE TITLE
Has this project been abandoned?

### ISSUE THREAD

**Issue by vitobotta** [randy-girard/app_perf#59]
Hi! I came thinking of trying this project again after a while but it seems it's not been updated recently. Have you abandoned the project? Thanks!

**Comment by thien0291** [comment]
It looks very promising... But long time no updates

**Comment by itay-grudev** [comment]
If 10 companies paid 50% of their Scout and Skylight bills to the author, it'd be a more successful project.

**Comment by randy-girard** [comment]
Yea sorry guys. At the moment its on the back burner. I would love to get back into it, I just need to find the time.

**Comment by itay-grudev** [comment]
@randy-girard Enroll to [GitHub Sponsors](https://github.com/sponsors) and add a [sponsor link](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository) to your repository.

**Comment by randy-girard** [comment]
Thanks. I am going to take a look at that. Might be the motivation i need :)

**Comment by vitobotta** [comment]
@randy-girard I do think that you could make this into a product. When I tried it - and it was a while ago - it was already pretty good (albeit it added significant latency at the time).

**Comment by randy-girard** [comment]
Yea. I have toyed around with the idea from starting from scratch using what I have learned from the project to build something new. This was always kind of like a proof of concept for me to understand how everything would work.

**Comment by vitobotta** [comment]
> Yea. I have toyed around with the idea from starting from scratch using what I have learned from the project to build something new. This was always kind of like a proof of concept for me to understand how everything would work.

I think that if you started it from scratch from a clean slate I could contribute more easily. I would be happy to do that :)

**Comment by thien0291** [comment]
I find out there are several open APMs... some of them are very popular & well-known. One of that is [ElasticSearch APM](https://github.com/elastic/apm-server). Personally, I love RoR but I have to say `golang` does a better job on `concurrency` stuffs which is necessary to collect data from multiple sources. ElasticSearch is good to index, scale horizontally. So, if you want to start it again @randy-girard, I suggest to know what make `app_perf` useful.

By the way. I went through your all related projects... You did a very good job!!!!!

**Comment by randy-girard** [comment]
So is the suggestion to build this as a GEM that you add to small to mid RoR projects only, and focus it that way?

**Comment by thien0291** [comment]
> So is the suggestion to build this as a GEM that you add to small to mid RoR projects only, and focus it that way?

Yes. I have that thought after seeing around... That may not be the best idea but it  can help me/us to not depend on any external services. What do you think about that 🤔?

**Comment by randy-girard** [comment]
The challenge is in order for this to be maintained long term, funding is a helpful thing. With small projects, there probably wont be much funding. My original idea was to provide support licenses to larger companies/or for advanced features to keep it funded

**Comment by thien0291** [comment]
> The challenge is in order for this to be maintained long term, funding is a helpful thing. With small projects, there probably wont be much funding. My original idea was to provide support licenses to larger companies/or for advanced features to keep it funded

You're right. I have to say that it is hard to get funds for that kind of project. But, with a hundred APM (free / open source / licensed)  out there, it is hard to compete too. That is why I start with `useful` :). Rails has a ton of issues itself (memory bloating/leak, object allocation, CPU overloading, poor graphql support,...) . Therefore, if you focus on a  niche market, you have chances to commercial it.


Lets say, I'm working with graphql and have ten thousands issues with that. No end to end solutions that fit with my needs (or for now, at least). It would save me a lot of time if there is a solution for it. 

All the APM platforms out there get hyped with micro services or many complicated solutions... But that complication makes me scare
